ambry-clustermap/src/main/java/com/github/ambry/clustermap/PartitionLayout.java

```diff
@@ -268,6 +269,9 @@ public void addNewReplicas(Partition partition, List<Disk> disks) {
    */
   public Partition getPartition(InputStream stream) throws IOException {
     byte[] partitionBytes = Partition.readPartitionBytesFromStream(stream);
+    logger.info("Partition bytes from stream: " + Arrays.toString(partitionBytes));
```

Comment:
This logging probably won't help exactly since the bytes read from stream will include the version number and the partition number concatenated